android/src/main/kotlin/com/builttoroam/devicecalendar/CalendarDelegate.kt

```diff
@@ -259,7 +262,6 @@ class CalendarDelegate : PluginRegistry.RequestPermissionsResultListener {
         finishWithSuccess(calendarId.toString(), pendingChannelResult)
     }
 
-    @SuppressLint("MissingPermission")
```

Comment:
Is this suppression no longer required?

Reply:
Yes I don't see any warnings in the method after removing it